app/jsx/components/TabMetricsComp.jsx

```diff
@@ -3,6 +3,15 @@
 import React from 'react'
 
 class TabMetricsComp extends React.Component {
+  componentWillMount() {
+    if (!(typeof document === "undefined")) {
```

Comment:
@amardesi For this library in this context, the way you've done it seems best. It's an external library, and they want it included a certain way, and it's browser-side only, and it is only used on a tab that is seldom clicked (I would think).